build.sbt

```diff
@@ -174,7 +174,8 @@ lazy val commonSettings = Seq(
   publishTo := Some(Resolver.file("Unused transient repository", file("target/unusedrepo"))),
   Compile / packageDoc / publishArtifact := false,
   Compile / packageSrc / publishArtifact := false,
-  addCompilerPlugin("org.typelevel" %% "kind-projector" % "0.13.2" cross CrossVersion.full),
+  addCompilerPlugin("com.olegpy"    %% "better-monadic-for" % "0.3.1"),
+  addCompilerPlugin("org.typelevel" %% "kind-projector"     % "0.13.2" cross CrossVersion.full),
```

Comment:
Is this `CrossVersion` setting needed? I thougt this is needed sometimes when compiling for different scala versions, but this might be wrong!

Reply:
Good catch. I took it out.

Reply:
It looks like the flag is needed. I just took it out for a moment but the build failed. What is more, the docs also says it's kind of required: https://github.com/typelevel/kind-projector#using-the-plugin